### PR TITLE
Update Controller and Class Specs

### PR DESCRIPTION
**Pull request by michaelwehrley**
Update Controller and Class Specs

* Update specs for home controller for method `#getMore`.
* Add specs for Serializer `#to_serialize` functions that are used in feed.

[#100582464]

### CODE REVIEW

File: spec/javascripts/social_networking/home-controller_spec.js [hunk 1]
```diff
@@ -204,4 +209,80 @@ describe('HomeCtrl', function() {
...
+            return $('#jasmine_content');
+          }
+
+          function getFeed() {
+            return getFixture()
+                   .append('<div id="infinite-feed" infinite-scroll-disabled="false" />')
+                   .find('#infinite-feed');
+          }
+
+          it('', function() {
```
Comment (eschlange) [inline spec/javascripts/social_networking/home-controller_spec.js]: Missing test case description.

**Comment by eschlange** [comment]
Other than the missing test case description, it looks great!

**Comment by michaelwehrley** [comment]
whoops

**Comment by michaelwehrley** [comment]
@eschlange updated; thanks.

**Pull request merged**